services/dashboard.qmd

```diff
@@ -1,12 +1,62 @@
 ---
-title: Dashboard
-subtitle: A graphical interface for discovering Earth data science
+title: VEDA Dashboard
+subtitle: 10 Things to Know About the VEDA Dashboard
 ---
 
-The [VEDA Dashboard](https://www.earthdata.nasa.gov/dashboard/) is a publishing platform for results of scientific analysis of Earth data at NASA, 
-in the form of Dataset information pages ("Datasets") and stories ("Discoveries").
+The VEDA Dashboard is an open-source set of configurable tools and pages that enable you and your team to visualize, analyze, and communicate science data. Learn more about the VEDA Dashboard set of tools below. If you would like to learn more about how to create your own instance of the VEDA Dashboard, visit this page.
 
+### 1. Science teams need to present their research program, findings, and methods and data to their stakeholders, colleagues, and other audiences
+- Modern data-driven science is not only presented through papers, posters, and lectures to fellow scientists and through reports to decision makers, but also through up-to-date interactive online tools and blogs that are available directly to either audience.
+Web and cloud platforms can enable scientists to publish contents in these interactive ways.
+- Often, science programs will want to put their information into a specific context and make it discoverable via their name, which is also important for credibility and attribution.
+- VEDA UI helps science teams combine the two - program-specific websites with interactive data visualization and standardized access.
```

Comment:
I wonder it is more effective to use this sentence as a title since the current title is more about the generic problem description than the fact about VEDA dashboard?